cafe.rb

```diff
@@ -14,12 +16,14 @@
   { name: 'ホットサンド', price: '410' }
 ].freeze
 
+i = 0
+
 def take_order(menus)
   menus.each.with_index(1) do |menu, i|
     puts "(#{i})#{menu[:name]}: #{menu[:price]}円"
   end
   print '>'
-  order_number = gets.to_i
+  order_number = gets.to_i-1
```

Comment:
[NITS]
細かい話なのですが、`.to_i` と `- 1` の間はスペースを入れると読みやすくなります。
くっついていると境目がわかりづらくなっちゃうんですね。
なのでこういうときはスペースを入れるのが一般的なお作法になります。
```suggestion
order_number = gets.to_i - 1
```